C-Plus-Plus/cp/recursion_string_permutation.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+    Here I use recursion to find all permutation of a string.
+    This method is useful for small length's strings because its time complaxity is in terms of n!  
+    and for non repeated character in string.
+*/
+#include<bits/stdc++.h>
+using namespace std;
+
+void rec(string s , int left_index , int right_index){
```

Comment:
Use proper name to define function. (Function name is not understandable. What is `rec`).